modules/javafx.graphics/src/test/java/test/javafx/scene/effect/BoxBlur_properties_Test.java

```diff
@@ -28,26 +28,21 @@
 import static test.com.sun.javafx.test.TestHelper.box;
 
 import java.util.ArrayList;
-import java.util.Collection;
+import java.util.stream.Stream;
 
 import javafx.scene.Node;
 import javafx.scene.effect.BoxBlur;
 import javafx.scene.effect.ColorAdjust;
 import javafx.scene.shape.Rectangle;
 
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameters;
-
-import test.com.sun.javafx.test.BBoxComparator;
+import org.junit.jupiter.params.provider.Arguments;
 import test.com.sun.javafx.test.PropertiesTestBase;
+import test.com.sun.javafx.test.BBoxComparator;
 
-@RunWith(Parameterized.class)
 public final class BoxBlur_properties_Test extends PropertiesTestBase {
 
-    @Parameters
-    public static Collection data() {
-        ArrayList array = new ArrayList();
+    public static Stream<Arguments> data() {
```

Comment:
you _could_ return the `List<Arguments>` directly

Reply:
Following above comments, I'll leave it as `Stream<Arguments>`